lib/clearance/session.rb

```diff
@@ -77,7 +79,14 @@ def sign_out
       end
 
       @current_user = nil
-      cookies.delete remember_token_cookie
+      if Clearance.configuration.cookie_domain.present?
+        cookies.delete(
+          remember_token_cookie,
+          domain: Clearance.configuration.cookie_domain,
+        )
```

Comment:
The API states that when you want to delete a cookie with a custom domain, you must pass that same domain option to it.

Reply:
Is there an adverse effect to just always passing the domain here? If the value is nil ... what happens?

Reply:
That's a good point, I'll investigate.